deploy/docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+
+services:
+  cache:
+    depends_on:
+      - tfserving
+    image: tfservingcache
+    build: 
+        context: ./../..
+        dockerfile: build/docker/Dockerfile
+    ports:
+      - "8093:8093"
+      - "8100:8100"
```

Comment:
When there is only one service running, it is only necessary to expose the ports of the cache and not the proxy, since we will always hit the right node. 
```suggestion
```

Reply:
If I understood correctly, currently, Prometheus metrics are provided through a "proxy" port so I expose it to have the ability to test it also. 
Anyway, it's can be done later when the integration tests around metrics will be written.

Reply:
Good catch! You're right.